src/components/Slider.js

```diff
@@ -1,39 +1,25 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-class Slider extends React.Component {
-  state = {
-    sliderVal: 5
-  };
-
-  static getDerivedStateFromProps(nextProps, prevState) {
-    // if the keyboards are changing the volume, adjust slider accordingly
-    if (nextProps.currentVolume * 10 !== prevState.sliderVal) {
-      return { sliderVal: nextProps.currentVolume * 10 };
-    }
-    return null;
-  }
-
-  handleSliderChange(e) {
+function Slider({ currentVolume, setTargetVolume }) {
```

Comment:
Nit: Could we make this an array function, since we are using ES6 in other places 😄

Reply:
other than that it is LGMT 👍

Reply:
Thank you both for the review :)

Here is the requested change: ecd737a5acb1ebdc751b63c51717c93df0eae0f2